finetune/lora.py

```diff
@@ -50,7 +50,8 @@ def main(
     out_dir: str = "out/lora/alpaca",
 ):
 
-    fabric = L.Fabric(accelerator="cuda", devices=1, precision="bf16-true")
+    devices = torch.cuda.device_count()
+    fabric = L.Fabric(accelerator="cuda", devices=devices, precision="bf16-true")
```

Comment:
```suggestion
    fabric = L.Fabric(accelerator="cuda", devices="auto", precision="bf16-true")
```

Reply:
@Abecid Did multi-gpu training work with this script and how many did you use? Is the loss convergence comparable to single gpu training?